templates/ossec/config/rubber/rubber-ossec.yml

```diff
@@ -0,0 +1,21 @@
+ossec_version: 2.8.1
+
+#http://ossec-docs.readthedocs.org/en/latest/manual/output/syslog-output.html#examples
+ossec_send_to_syslog: true
+ossec_syslog_server_address: 10.10.10.10
```

Comment:
should this reference the ossec_master role?  Does this template have the setup to do the master/agent setups?